src/main/java/com/sparta/vicky/jwt/jwtUtil.java

```diff
@@ -0,0 +1,108 @@
+package com.sparta.vicky.jwt;
+
+import io.jsonwebtoken.*;
+import io.jsonwebtoken.security.Keys;
+import jakarta.annotation.PostConstruct;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+import java.security.Key;
+import java.util.Base64;
+import java.util.Date;
+
+@Slf4j
+@Component
+public class jwtUtil {
+    // Header KEY 값
+    public static final String AUTHORIZATION_ACCESS_HEADER = "Authorization_Access";
+    public static final String AUTHORIZATION_REFRESH_HEADER = "Authorization_Refresh";
+    // Token 식별자
+    public static final String BEARER_PREFIX = "Bearer ";
+
+    @Value("${jwt.secret.key}") // Base64 Encode 한 SecretKey
+    private String secretKey;
+    private Key key;
+    private final SignatureAlgorithm signatureAlgorithm = SignatureAlgorithm.HS256;
+
+    @PostConstruct
+    public void init() {
+        byte[] bytes = Base64.getDecoder().decode(secretKey);
+        key = Keys.hmacShaKeyFor(bytes);
+    }
+
+    // Access 토큰 생성
+    public String createAccessToken(String username) {
+        Date date = new Date();
+
+        // 60분 토큰 만료시간
+        long ACCESS_TOKEN_TIME = 60 * 60 * 1000L;
+
+        return BEARER_PREFIX +
+                Jwts.builder()
+                        .setSubject(username) // 사용자 식별자값(ID)
+                        .setExpiration(new Date(date.getTime() + ACCESS_TOKEN_TIME)) // 만료 시간
+                        .setIssuedAt(date) // 발급일
+                        .signWith(key, signatureAlgorithm) // 암호화 알고리즘
+                        .compact();
+    }
+
+    // Refresh 토근 생성
+    public String createRefreshToken(String username) {
+        Date date = new Date();
+
+        //토큰 만료시간 일주일
+        long REFRESH_TOKEN_TIME = 7 * 60 * 60 * 60 * 1000L;
```

Comment:
요구사항에 refresh token 만료기간이 2주라고 되어있어요.
```suggestion
        // 토큰 만료시간 (2주)
        long REFRESH_TOKEN_TIME = 14 * 24 * 60 * 60 * 1000L;